upstairs/src/lib.rs

```diff
@@ -1632,6 +1634,8 @@ pub struct Arg {
     pub ds_extents_repaired: [usize; 3],
     /// Times we have live confirmed  an extent on this downstairs.
     pub ds_extents_confirmed: [usize; 3],
+    /// If in Live Repair, the current extent we are repairing.
+    pub ds_extent_limit: usize,
```

Comment:
How close are we to the `strsize=1k` DTrace limit?

Reply:
Looks like we are currently at 843, at least that is how much JSON I get.

Reply:
Great, that's not too close to the limit